WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -43,9 +43,6 @@ extension UserDefaults {
         // AI prompt tone
         case aiPromptTone
 
-        // Celebration view after Blaze campaign creation
-        case hasDisplayedTipAfterBlazeCampaignCreation
```

Comment:
IMO, we cannot guarantee that users install every app update, so we can't be sure that this process will eliminate the edge case entirely. The best we can do is create unique names for the user defaults, like assigning the origin app version for the key to avoid duplications.

Reply:
> IMO, we cannot guarantee that users install every app update, so we can't be sure that this process will eliminate the edge case entirely. 

I agree. In step 4, we only add a comment to mark the case as deprecated. But the `case` is not removed ever. I thought it would cover the case of users skipping app updates.

> The best we can do is create unique names for the user defaults, like assigning the origin app version to the keys to avoid duplications.

This sounds great. 👍